book/website/communication/citable/citable-steps.md

```diff
@@ -0,0 +1,72 @@
+(rr-credit-make)=
+# Steps for Making Research Objects Citable
+
+There are many reasons why authors don't cite the data and software that they use, but one of the biggest ones is that it's not clear how.
+You can go a long way to reducing this barrier by following a few steps to make it as easy as possible.
+
+(rr-credit-make-object)=
+## 1. Identify Your Research Objects
+
+The first step is to ensure that you have something worth citing.
+Practising open research isn't essential to get credit for your data or software, but it makes it much easier for others to build on your work in a way that acknowledges your contribution.
+There is a growing body of evidence that shows open research tends to be cited more than non-open research of equivalent quality and significance.
+
+<!-- TODO: Cite relevant paper for this (Piwowar et al 2013?) -->
+
+```{note}
+You can learn more about the different types of research objects in the chapters {ref}`making your research open<rr-open>` and {ref}`making your research FAIR<rr-rdm>`.
+```
+
+(rr-credit-make-publish)=
+## 2. Publish your work online
+
+Every online publication is attached to a unique identifier that is used for citing them.
```

Comment:
Think this needs to be qualified so it fits with the sentence that follows - not everything published online gets a unique identifier but it is important that research objects have a DOI if they're being published online